services/essentials.yaml

```diff
@@ -6,18 +6,19 @@ metadata:
   annotations:
 spec:
   core:
-    ref: master
-    # version: 1.0
+    version: '2.135'
 
   plugins:
-    - groupId: 'org.jenkins-ci.plugins'
+    - groupId: 'io.jenkins'
       artifactId: 'configuration-as-code'
-      # TODO overridden
-      version: '0.1-alpha'
+      version: '0.11-alpha-rc362.942711740b07'
+    - groupId: 'io.jenkins.plugins'
+      artifactId: 'essentials'
+      version: '0.3-rc20.a3f76cdd1348'
 
     - groupId: 'io.jenkins.blueocean'
       artifactId: 'blueocean'
-      version: '1.4.2'
+      version: '1.7.1'
```

Comment:
Last I saw, this pulled latest dependencies from the update center, and so was quite useless. Still true?

Reply:
Well, when I worked on my last PR (#144), I noticed that the transitive dependencies were taken from the latest version on the UC. Since the `blueocean` plugin itself is just an aggregator, that made the actual version you specify here pretty much irrelevant—running the ingestion script would always pick up the latest version of every component. I have not tried the rewritten script locally, but I had another comment to the effect that you should never use a nondeterministic version for a transitive dependency: you should use least upper bounds.